static/scss/detail/learning-outcomes.scss

```diff
@@ -0,0 +1,56 @@
+// sass-lint:disable mixins-before-declarations
+
+.learning-outcomes-block {
+  background: white url('#{$static-path}/images/bg-access.png') repeat-x 0 0;
+  padding: 50px 0;
+
+  .heading-container {
+    margin: 0 auto;
+    max-width: 1000px;
+    text-align: center;
+    font-size: 28px;
+    line-height: 38px;
+    font-weight: 500;
+  }
+
+  h1 {
+    max-width: 700px;
+    margin: 0 auto 20px;
+    font-size: 40px;
+    line-height: 44px;
+    color: $red;
+    font-weight: 600;
+  }
+
+  p {
```

Comment:
With my suggestion above regarding changing the `p` to an `h3`, this shouldn't be necessary. Can you try the following changes?:

- Add a style definition for `h1, h3`: `margin-top: 0; margin-bottom: 30px;`
- If we really do need the heading to be rich text, add a style to remove margins and padding for `h1 p`

Reply:
I have added a new file `common.scss` file under `detail` folder that would be used for our own `.scss` files and added the following definition. 

```
body {
  line-height: 28px;
  font-size: 20px;
  font-weight: 500;
}

h1 {
  font-size: 40px;
  line-height: 44px;
  font-weight: 600;
}

h3 {
  font-size: 28px;
  line-height: 38px;
  font-weight: 500;
} 
```